.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: python
 sudo: false
-install: pip install tox coveralls --use-mirrors
+before_install: if [ "$JYTHON" == "true" ]; then travis_scripts/install_jython2.7.sh; fi
```

Comment:
These single-line-ifs all over the place are so ugly. If someone has thoughts on how to do this better, would love to hear them!

I could pull the ifs and all into `travis_scripts` perhaps.